README.md

```diff
@@ -14,37 +14,64 @@ Information for package authors can be found at
 
 It's recommended to use `stack`: <http://docs.haskellstack.org>.
 
+### Build
+
 To build in development mode:
 
 ```
 $ stack build
 ```
 
-To run the server:
+To build in production mode:
 
 ```
-$ stack exec pursuit
+$ stack build --flag pursuit:-dev
 ```
 
-If you have content in your data directory
-(see [Database structure](#database-structure)) then the database will
-be regenerated before the server starts listening - this can take a short time
-depending on how much data you have.  The site should then be available at
-`http://localhost:3000`.
+### Develop
 
-To build in production mode:
+To iterate quickly during development, you can use `ghci`:
 
 ```
-$ stack build --flag pursuit:-dev
+$ stack ghci
 ```
 
-## Database structure
+Once the REPL has loaded, you can reload the code and then update the web server:
+
+```
+> :l DevelMain
+> update
+```
+
+### Web server
+
+To run the web server on <http://localhost:3000>:
+
+```
+$ stack exec pursuit
+```
+
+Make sure you have content in your `data/verified` directory (see
+[Database](#database)), otherwise you will not be able to browse any packages. The
+database will be regenerated from this data source before the server starts
+listening; this can take a short time depending on how much data you have.
+
+## Database
 
 Pursuit currently uses the filesystem as a database, since it requires no setup
 and it makes it easy to use Git and GitHub for backing up. The data directory
 is set via an environment variable (see [Configuration](#configuration)).
 
-The structure is as follows:
+If you need some sample packages to work with, you can clone the
+[pursuit-backups][pursuit-backups] repo and copy the packages you want to the
+`verified/` directory. This is more convenient than manually uploading each
+package.
+
+[pursuit-backups]: https://github.com/purescript/pursuit-backups
+
+## Database structure
+
+The database lives in the `data` directory, its structure is as follows:
```

Comment:
I think this needs to be amended slightly as it seems to contradict the sentence in a previous paragraph:

> The data directory is set via an environment variable (see [Configuration](#configuration)).